src/stackdriver-nozzle/nozzle/log_sink.go

```diff
@@ -84,10 +84,10 @@ func (ls *logSink) parseEnvelope(envelope *events.Envelope) messages.Log {
 			// to remain consistent with the protobuf.
 			logMessageMap["message_type"] = logMessage.GetMessageType().String()
 			severity = parseSeverity(logMessage.GetMessageType())
-			logMessageMap["message"] = message
+			delete(logMessageMap, "message")
```

Comment:
@pivotalsquid Can you do some rearranging? Can you move the `payload["message"] = message` line before the delete?

Then, add a comment to the delete line which states the delete is removing the duplicate message 

This would make the reasoning for the delete more obvious.

Reply:
Sure, I can make that change.

Reply:
Change complete.